aptos-move/aptos-debugger/src/lib.rs

```diff
@@ -23,6 +23,7 @@ use aptos_vm::{
 };
 use move_binary_format::errors::VMResult;
 use std::{path::Path, sync::Arc};
+use aptos_types::on_chain_config::{FeatureFlag, Features};
```

Comment:
you're going to be yelled at for not using `cargo +nightly fmt` or `pre-commit run --all-files`

Reply:
somehow `pre-commit run --all-files` is hooked in `git commit` for me.

Reply:
Fixed.